src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -704,6 +709,9 @@ def test_minigraph_bgp_voq_chassis_peer(self):
     def test_minigraph_sub_port_interfaces(self, check_stderr=True):
         self.verify_sub_intf(check_stderr=check_stderr)
 
+    def test_minigraph_sub_port_intf_resource_non_backend_tor(self, check_stderr=True):
+        self.verify_sub_intf_non_backend_tor(graph_file=self.sample_resource_graph, check_stderr=check_stderr)
```

Comment:
Why is it named "sub intf"?

Reply:
I use `test_minigraph_sub_port_intf_resource_type` implementation as a reference.